server/src/main/java/org/elasticsearch/index/engine/NoOpEngine.java

```diff
@@ -67,6 +70,7 @@
                     throw new IllegalArgumentException("Expected 0 translog operations but there were " + nbOperations);
                 }
             }
+            this.docsStats = docsStats(getLastCommittedSegmentInfos());
```

Comment:
any reason not pull this up to ReadOnlyEngine?

Reply:
++ lets do that

Reply:
Sure